tests/thread_safe_logger/run_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+pushd ../.. > /dev/null 2>&1
+
+output="$(./build/bin/thread_safe_logger -thread_count 4 -run_time_ms 1000 --logall 2>&1)";
```

Comment:
Is a run time of 1000 really enough? Have we enough insurance that threads don't just finish before the other threads are created?